crucible.mjs

```diff
@@ -259,6 +260,57 @@ Hooks.once("init", async function() {
     default: false
   });
 
+  const fields = foundry.data.fields;
+  function fieldForType(type) {
+    const getPacks = (type) => {
+      const potentialPacks = {};
+      for (const pack of game.packs) {
+        if (pack.metadata.type !== "Item") continue;
+        for (const item of pack.index) {
+          if (item.type === type) {
+            potentialPacks[pack.metadata.id] = `${pack.metadata.label} (${pack.metadata.id})`;
```

Comment:
Stretch goal, but very nice-to-have:

When preparing the `choices` for packs, it would be a nice UX to organize them by `group` (see the `FormSelectOption` typedef) where the `group` would be the system or module that provides them. 

It would also be ideal to specify the `groups` parameter of `SelectInputConfig` so we can cause system-provided packs to always appear at the top of the list, followed second by world-level packs, and then followed by module-provided packs in alphabetical order.